app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+  def new
+    @admin = Admin.new
+  end
+
+  def create
+    admin = Admin.new(admin_params)
+    if admin.save
+      flash[:notice] = "You are successfully logged in"
+      redirect_to admins_dashboard_index_path
```

Comment:
Same as above. The page that will allow the admin to see all of the companies that have signed up.